### PR TITLE
AADRemoteNetwork

### PR DESCRIPTION
**Pull request by PiyushDub**
#### Pull Request (PR) description
AADRemoteNetwork Initial release

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

File: Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/3-Remove.ps1 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/3-Remove.ps1]: Let's completely remove the file in this case

File: Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/1-Create.ps1 [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/1-Create.ps1]: Let's completely remove the file in this case

**Review by NikCharlebois — CHANGES_REQUESTED** [review]
Please remove the 2 empty example files.

File: Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/3-Remove.ps1 (hunk 1 above)
Comment (PiyushDub) [reply Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/3-Remove.ps1]: updated

File: Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/1-Create.ps1 (hunk 2 above)
Comment (PiyushDub) [reply Modules/Microsoft365DSC/Examples/Resources/AADRemoteNetwork/1-Create.ps1]: updated

**Review by NikCharlebois — APPROVED** [review]

**Pull request merged**